pathod/language/generators.py

```diff
@@ -55,12 +55,12 @@ def __len__(self):
         return self.length
 
     def __getitem__(self, x):
-        return random.choice(DATATYPES[self.dtype])
+        return random.choice(DATATYPES[self.dtype]).encode()
```

Comment:
We could, but I think we should tackle the problem at the source and fix DATATYPES directly.